README.md

```diff
@@ -3,11 +3,12 @@ Operator that manages Scheduling, Scale and Performance addons for [KubeVirt](ht
 
 ## Functionality
 
-The operator deploys and manages resources needed by these four components:
+The operator deploys and manages resources needed by these components:
 
 - [Template Validator](https://github.com/kubevirt/ssp-operator/tree/master/internal/template-validator) (You can read more about it here: [Template Validator old repository](https://github.com/kubevirt/kubevirt-template-validator))
 - [Common Templates Bundle](https://github.com/kubevirt/common-templates)
-- Metrics rules - Currently it is only a single Prometheus rule containing the count of all running VMs.
+- DataVolumes for templates - Images are auto-imported to the DataVolumes as specified in the SSP CR.
```

Comment:
`DataImportCrons and DataSources for Templates - Images are auto-imported via DataImportCrons to the DataSources as specified in the SSP CR.`?